hypha/templates/base.html

```diff
@@ -123,6 +123,45 @@
         </script>
         <!-- htmx end -->
 
+        <!-- choices select -->
+        <script type="module">
+            import Choices from "{% static 'js/esm/choices.js-10-2-0.js' %}";
+
+            htmx.onLoad(function() {
+                const selectElements = document.querySelectorAll("[data-js-choices]");
+
+                selectElements.forEach((selectElement) => {
+                    if (!selectElement.choicesInstance) {
+                        const choicesInstance = new Choices(selectElement, {
+                            shouldSort: false,
+                            allowHTML: true,
+                            removeItemButton: true,
+                        });
+                        selectElement.choicesInstance = choicesInstance;
+                    };
+                });
+
+                const choicesElements = document.querySelectorAll(".choices__input--cloned")
+                choicesElements.forEach((choiceElement) => {
+                    const inputPlaceholder = choiceElement.getAttribute("placeholder");
+                    // Get the computed min-width of the input element otherwise it reset to 1
+                    const minWidth = window.getComputedStyle(choiceElement).getPropertyValue("min-width");
+                    choiceElement.addEventListener("focus", () => {
+                        if (choiceElement) {
+                            choiceElement.placeholder = "Search..."
```

Comment:
Need to add something like `choiceElement.style.minWidth = "10ch";` or the search placeholder gets cut of if the original placeholder is shorter.

Reply:
Right, 7ch looks enough for the search placeholder.